biosignals.py

```diff
@@ -0,0 +1,74 @@
+import platform
```

Comment:
THis new class is meant to work with the plux api (should have been included in the repo, hopefully), which controls the biosignal plux device, which I was not expecting to have to include here since I already have a program that deals with that, but anyways. Its requirements are diferent than the other api's, so I made a new class which I can integrate back into our recording handler, once they are similar enough.